frontend/src/app/shared/components/autocompleter/op-autocompleter/op-autocompleter.spec.ts

```diff
@@ -151,9 +151,6 @@ describe('autocompleter', () => {
       const inputElement = inputDebugElement.nativeElement as HTMLInputElement;
 
       fixture.detectChanges();
-      tick();
-      expect(getOptionsFnSpy).not.toHaveBeenCalled();
-      tick(50);
```

Comment:
@myabc Can I ask you kindly to check whether this testcase is still valid. The [component has changed a bit](https://github.com/opf/openproject/pull/18530/files#diff-fd400a45118bce80699f5aa1d380d8fe8ca08e84a4dafc092231d68bf5d77cb8), which causes the spec to fail. Thank you!